### PR TITLE
fixing command if config-file isn't passed

### PR DESCRIPTION
**Pull request by kruxErinWillingham**
fixing command if config-file isn't passed

### CODE REVIEW

File: krux_marathon_api/cli.py [hunk 1]
```diff
@@ -33,9 +33,12 @@ def __init__(self):
...
         ### Handles files passed via i.e. ~/some-link.json and it will translate
         ### to the proper full location
-        self.marathon_config_file = os.path.realpath(
-            os.path.expanduser(self.args.config_file)
-        )
+        if self.args.config_file:
+            self.marathon_config_file = os.path.realpath(
+                os.path.expanduser(self.args.config_file)
+            )
+        else:
```
Comment (thekad) [inline krux_marathon_api/cli.py]: I don't think you need the else clause here, self.args.config_file will be None thus working as expected?

File: krux_marathon_api/cli.py [hunk 2]
```diff
@@ -33,9 +33,12 @@ def __init__(self):
         self.marathon_list_apps = self.args.list_apps
         ### Handles files passed via i.e. ~/some-link.json and it will translate
```
Comment (hanpeter) [inline krux_marathon_api/cli.py]: Nit: Let's move this comment into the `if` clause.

File: krux_marathon_api/cli.py [hunk 3]
```diff
@@ -33,9 +33,12 @@ def __init__(self):
...
         ### to the proper full location
-        self.marathon_config_file = os.path.realpath(
-            os.path.expanduser(self.args.config_file)
-        )
+        if self.args.config_file:
+            self.marathon_config_file = os.path.realpath(
+                os.path.expanduser(self.args.config_file)
+            )
+        else:
+            self.marathon_config_file = self.args.config_file
```
Comment (hanpeter) [inline krux_marathon_api/cli.py]: `False` doesn't sound like a good default value for config file. Wouldn't `None` be better?

File: krux_marathon_api/cli.py (hunk 1 above)
Comment (hanpeter) [inline krux_marathon_api/cli.py]: @thekad, the default value for `--config-file` is `False`.

Also, I would argue it is more readable to still use the `else` here, though I am far from being married to that view.

Comment (kruxErinWillingham) [inline krux_marathon_api/cli.py]: line 109's variable wouldn't be set w/o the else.

**Pull request merged**